File: src/test/resources/integration_tests/clojure/cp_test.clj
(ns integration-tests.clojure.cp-test
  (:require [vertx.core :as core]
            [vertx.eventbus :as eb]
            [vertx.logging :as log]
            [vertx.testtools :as tt])
  (:import [java.net URL URLClassLoader]))



(defn test-ture []
  (let [urls (-> (Thread/currentThread)
               .getContextClassLoader
               .getURLs)]
    (log/info (alength urls))
    (doseq [url (seq urls)]
      (log/info url)))
  (log/info "................")
  (tt/assert= 1 1)
  (tt/test-complete))


(tt/start-tests)

File: src/test/java/cn/intellijoy/vertx/filetransfer/unit/CpTest.java
package cn.intellijoy.vertx.filetransfer.unit;

import java.net.URL;
import java.net.URLClassLoader;

import org.junit.Test;

public class CpTest {

  @Test
  public void t() {
    URL[] urls = ((URLClassLoader) (Thread.currentThread().getContextClassLoader())).getURLs();

    for (URL url : urls) {
      System.out.println(url);
    }
  }
}
